src/components/ui/navbar.tsx

```diff
@@ -0,0 +1,80 @@
+"use client";
+
+import React, { useState } from "react";
+import Link from "next/link";
+import Image from "next/image";
+
+const Navbar: React.FC = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleMenu = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <nav className="project-200-white border-b border-gray-200">
+      <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
+        <div className="flex justify-between items-center h-16">
+          {/* Logo Section */}
+          <div className="flex items-center">
+            <Link href="/">
+              <Image
+                src="/path-to-your-logo.png" // Replace with the correct logo path
+                alt="Project 200 Logo"
+                width={50}
+                height={50}
+                className="h-10 w-auto"
+              />
+              <h3 className="text-2xl">Project 200</h3>
```

Comment:
We can remove the text since we are only using the png logo. See:
![Screenshot 2024-12-10 at 6 59 24 AM](https://github.com/user-attachments/assets/b4a264fa-3110-4928-87c6-2a7d153a68d8)